scripts/peak2bw.py

```diff
@@ -49,57 +30,36 @@
 _logger.addHandler(_handler)
 
 
-def parse_args():
-    """Parse command line arguments.
+def peak2bw(input_file, sizesfile, out_dir):
```

Comment:
same, how about moving this over to `atacworks/io`?

Reply:
See explanation for bw2h5